vnext/.flowconfig

```diff
@@ -135,8 +125,6 @@ suppress_type=$FlowFixMeEmpty
 
 suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe\\($\\|[^(]\\|(\\(<VERSION>\\)? *\\(site=[a-z,_]*react_native\\(_ios\\)?_\\(oss\\|fb\\)[a-z,_]*\\)?)\\)
 suppress_comment=\\(.\\|\n\\)*\\$FlowIssue\\((\\(<VERSION>\\)? *\\(site=[a-z,_]*react_native\\(_ios\\)?_\\(oss\\|fb\\)[a-z,_]*\\)?)\\)?:? #[0-9]+
-suppress_comment=\\(.\\|\n\\)*\\$FlowFixMe\\($\\|[^(]\\|(\\(<VERSION>\\)? *\\(site=[a-z,_]*react_native\\(_android\\)?_\\(oss\\|fb\\)[a-z,_]*\\)?)\\)
```

Comment:
@acoates-ms could you revert this part? We derive from some Android files as well and at least in 0.62 needed to account for suppressions for both platforms.

Reply:
I need to apply the same changes to the win32 flowconfig anyway, so I'm going to go ahead and add this back to both. Let me know if you disagree and I can revert it,

Reply:
Really we should get the flowconfig in the override file.  I messed this up trying to sync our flowconfig with upstreams.

Reply:
Yeah, I agree. Have an item for tracking doing automatic integration. We should be able to use the existing infrastructure.